index.d.ts

```diff
@@ -1,11 +1,33 @@
-import {Options} from 'filenamify';
+import {Options as FilenamifyOptions} from 'filenamify';
```

Comment:
Shouldn't this use the CommonJS import syntax?

Reply:
We can import interfaces the way we imported them before. This doesn't depend on default exports.